### DISCUSSION TITLE
How to extend my network to the shed?

### DISCUSSION THREAD

**Discussion by jhkuperus** [Koenkk/zigbee2mqtt#7624] [General]
I've been using zigbee2mqtt with Home Assistant for a couple of years now and I want to extend my coverage to the shed / sitting area in the back of the garden. There are two way I can see going forward: 

1. Place a series of zigbee-capable lights along the border of my garden until the lights in the back are in range. This seems like the silly way to do it, but also a pretty good fallback.
2. Place another Raspberry Pi in the shed with a Zigbee antenna. There is WiFi coverage and electricity there, so this seems easy enough, but I have a question about it:

* Can I connect a secondary (remote) CC-something antenna to the same mqtt instance?
* If so, what firmware should I flash the CC-stick with?

**Comment by Koenkk** [comment]
2 is simply just another zigbee2mqtt network, only make sure you use another `base_topic` (https://www.zigbee2mqtt.io/information/configuration.html)

**Reply by jhkuperus** [reply]
Okay, so then I flash the CC-stick with Coordinator-firmware, set a different base-topic, point it at Home Assistants broker and simply pair the lights in the shed with that stick and I'm done? Cool. Thanks!

**Reply by Koenkk** [reply]
Exactly! (but if possible, try to extend your existing network using a router, e.g. https://www.zigbee2mqtt.io/how_tos/how_to_create_a_cc2530_router.html, this will prevent the more complicated setup of having 2 z2m instances, 2 Pi's).

**Comment by arfond** [comment]
I had the same need - connecting lights and sensors in the shed to the house.

I made up a router and placed it in a weatherproof enclosure, and put it in a safe spot in the garden next to a power socket. It's about half way between the house and garden.

Works perfectly, all on one network.
